codegen/src/lib.rs

```diff
@@ -167,8 +167,16 @@ impl CodegenBuilder {
         &mut self,
```

Comment:
We'd need to adjust the docs of this function a bit, the warning section specifically